Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,126 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Reflection.Metadata;
+using System.Runtime.CompilerServices;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    class Lesson3
+    {
+        public static float IsNumber(string input)
+        {
+            var isEmpty = String.IsNullOrEmpty(input);
+           
+            if (isEmpty) return 0f;
+            
+            if ((input) == "0") return 0f;
```

Comment:
This is not needed, because Parse part covers parsing "0" as well.

Reply:
aaah i get it